core/src/main/groovy/io/snyk/plugins/artifactory/snykSecurityPlugin.properties

```diff
@@ -42,3 +42,12 @@ snyk.scanner.vulnerability.threshold=low
 # Global threshold for license issues. Valid values include "none", "low", "medium", "high", "critical".
 # Default value: "low"
 snyk.scanner.license.threshold=low
+# Enable Maven repository scanning.
+# Default value: "true"
+snyk.scanner.packageType.maven=true
+# Enable NPM repository scanning.
+# Default value: "true"
+snyk.scanner.packageType.npm=true
+# Enable PyPi repository scanning.
+# Default value: "true"
+snyk.scanner.packageType.pypi=true
```

Comment:
I would disable it by default for now. Python Test API is currently too slow to be used in the Snyk Artifactory Plugin

Reply:
Done.